qemu/tests/cfg/in_place_upgrade.cfg

```diff
@@ -59,7 +59,9 @@
         - without_rhsm_rhel8_to_rhel9:
             rhsm_type = "no_rhsm"
             pre_upgrade_no_rhsm = "leapp preupgrade --debug --no-rhsm"
-            process_upgrade_no_rhsm = "leapp upgrade --debug --no-rhsm"
+            process_upgrade_no_rhsm = "export LEAPP_UNSUPPORTED=1;"
+            process_upgrade_no_rhsm += "export LEAPP_DEVEL_SKIP_CHECK_OS_RELEASE=1;"
+            process_upgrade_no_rhsm += "leapp upgrade --debug --no-rhsm"
```

Comment:
I think LEAPP_xxx will only be used once, so not need to export them, just use env variables and pass to the command line, like in my previous comments.